src/pages/Daeyoung/Login/DaeyoungLogin.js

```diff
@@ -1,7 +1,92 @@
-import React from 'react';
 
-const DaeYoungLogin = props => {
-  return <div>Login</div>;
+import React , {useState} from "react"
+import "./DaeyoungLogin.scss" ;
+
+// import { useNavigate } from "react-router-dom";
+// import {Link} from "react-router-dom";
+
+
+function DaeyoungLogin(){
+
+// const navigate = useNavigate();
+
+// const goTomain = () =>{
+    // navigate("/main");
+// }
+
+
+const [idvalue,setid] = useState('');
+const [pwvalue,setpw] = useState('');
```

Comment:
setState하는 부분은 set + state명으로 만들기
```suggestion
const [idValue, setIdValue] = useState('');
const [pwValue,setPwValue] = useState('');
```